response_operations_ui/views/respondents.py

```diff
@@ -0,0 +1,42 @@
+import logging
+
+from flask import Blueprint, redirect, render_template, request, url_for
+from flask_login import login_required
+from structlog import wrap_logger
+
+from response_operations_ui.controllers import contact_details_controller, respondent_controllers
+from response_operations_ui.forms import SearchForm
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+respondent_bp = Blueprint('respondent_bp', __name__,
+                          static_folder='static', template_folder='templates')
+
+
+@respondent_bp.route('/', methods=['GET', 'POST'])
+@login_required
+def respondent_search():
+    form = SearchForm()
+    breadcrumbs = [{"title": "Respondents"}]
+    response = None
+
+    if form.validate_on_submit():
+        email = request.form.get('query')
+
+        respondent = respondent_controllers.search_respondent_by_email(email)
+
+        if respondent.get('id'):
+            return redirect(url_for('respondent_bp.respondent_details', respondent_id=respondent['id']))
+        else:
+            response = respondent['Response']
+
+    return render_template('search-respondent.html', response=response, form=form, breadcrumbs=breadcrumbs)
+
+
+@respondent_bp.route('/respondent-details/<respondent_id>', methods=['GET'])
```

Comment:
We're missing breadcrumbs for this endpoint

Reply:
@andrewmil @gemmai95 In some places we've added a _build_breadcrumbs function to split it out, what's your opinion on this? Because at the moment we're doing 2 things, would be nice to have it all the same

Reply:
I seen this in the messaging stuff it does look a lot cleaner but atm I just added it into the function.

Reply:
If we can easily generalise it to work across the repo I'm a fan of it @dcdarrell9.
Agree we should have consistency though for sure